rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/data/DataUtility.java

```diff
@@ -283,6 +285,12 @@ public static FilteredStatistics toFilteredStatistics(FilteredStatisticsData job
                                       jobsReportData.getSuccessfulRate());
     }
 
+    public static Map<String, String> setSubmissionModeToGenericInfo(Map<String, String> genericInformation) {
+        Map<String, String> newGenericInformation = new LinkedHashMap<>(genericInformation);
```

Comment:
"scheduler-api" should be externalized as a constant, typically in SchedulerConstants